storage/innobase/handler/ha_innodb.cc

```diff
@@ -17540,15 +17544,27 @@ ib_logf(
 	switch(level) {
 	case IB_LOG_LEVEL_INFO:
 		sql_print_information("InnoDB: %s", str);
+#ifdef HAVE_SYSTEMD
+		sd_notifyf(0, "STATUS=InnoDB: Info: %s", str);
+#endif
```

Comment:
1. what happens with STATUS messages that are issued when the service is running?
they go to syslog?
2. Perhaps fatal/error messages are significant, but info/warning can be pretty verbose and annoying. see, for example, MDEV-8310 and MDEV-8303 (there are just two recent bugs that I remember)